### ISSUE TITLE
ionic start don't work

### ISSUE THREAD

**Issue by Slumber86** [ionic-team/ionic-cli#22]
When i try to start a new project i get the following errror:

```
$ ionic start HybridDemo sideMenu
Running start task...
Creating Ionic app in folder /Users/xxx/Desktop/HybridDemo based on sideMenu project

DOWNLOADING: https://github.com/driftyco/ionic-app-base/archive/master.zip

DOWNLOADING: https://github.com/driftyco/ionic-starter-sideMenu/archive/master.zip
Initializing cordova project.
Fetching plugin "org.apache.cordova.device" via plugin registry
Error: failed to fetch the plugin archive
    at /usr/local/lib/node_modules/cordova/node_modules/plugman/src/registry/registry.js:60:26
    at Request.cb [as _callback] (/usr/local/lib/node_modules/cordova/node_modules/plugman/src/registry/registry.js:251:9)
    at self.callback (/usr/local/lib/node_modules/cordova/node_modules/plugman/node_modules/request/index.js:148:22)
    at Request.EventEmitter.emit (events.js:117:20)
    at ClientRequest.self.clientErrorHandler (/usr/local/lib/node_modules/cordova/node_modules/plugman/node_modules/request/index.js:257:10)
    at ClientRequest.EventEmitter.emit (events.js:95:17)
    at Socket.socketErrorListener (http.js:1547:9)
    at Socket.EventEmitter.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:415:13)
ERROR: Unable to add plugins. Perhaps your version of Cordova is too old. Try updating (npm install -g cordova), removing this project folder, and trying again.
Exiting.
```

reproduced on OSX and windows 8. obiuvsly cordova and ionic are up-to date

**Comment by eddiclin** [comment]
@saamorim It works. ^_^

**Comment by sinedied** [comment]
Why not using the standard "http_proxy" and "https_proxy" env variables? Pretty much all osx/linux programs support these (including cordova, npm, git...)

**Comment by Najuru** [comment]
@sinedied  
Your solution worked for me..!!! Thanks ...

I set my proxy as below : 

C:\Users\naj>npm config set proxy http://myUserName:myPassword@111.111.111.6:8000

C:\Users\naj>npm config set https-proxy http://myUserName:myPassword@111.111.111.6:8000

and upadate the same in system environment variable  "http_proxy" and "https_proxy". Cloased the cmd prompt once and i opend again ,it worked well..!!!

**Comment by weitony111** [comment]
@vincesp My idea of doing all these stuff behind NTLM  proxy is that use **CNTLM** to convert your NTLM proxy into a local proxy server which does not require any usernames or passwords. It really makes my life a lot easier.

**Comment by tesnep** [comment]
Here's what I found in Win7.

Setting NPM proxy is not going work with this. I tried forwarding to fiddler by setting the npm proxy and https-proxy like
`npm set proxy=http://localhost:8888`
and fillder would not capture the request.

I then simply set the proxy like
`set proxy=http://localhost:8888` and fillder captured the request and started downloading it. 

I think CNTLM will work too. I usually switch between setting the actual proxy 
`http://username:password@domain:port`, fiddler and CNTLM. When none of these work I simply try to download the package or get it working on my home pc and copy over. Windows complains about too many characters in path but somehow after that it manages to work with the proxy.
